content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -342,7 +341,7 @@ You must use a `namespaceSelector` with `matchLabels` or `matchExpressions` to s
 namespaces based on their labels.
 {{< /note >}}
 
-## Targeting a Namespace by its name
+## Targeting a namespace by its name
```

Comment:
Title case for Namespace is appropriate here; we're talking about a thing that has `.metadata.labels` and a `.metadata.name`, so we're mostly framing it as the API kind.